### DISCUSSION TITLE
ExpressLRS for Video Transmission and Reception

### DISCUSSION THREAD

**Discussion by ziiw** [ExpressLRS/ExpressLRS#1123] [Q&A] [answered]
Hi everyone!

Maybe a dumb question, could ELRS be used for video transmission/reception? 
Just wondering if LORA and the tech behind ELRS could enhance what's happening in the VT/RX world :)

Have a nice day!

**Comment by AlessandroAU** [comment, marked as answer]
The data-rate achievable with lora is far too low for video

**Reply by Lalbabu2001** [reply]
Isn't there any hardware we can develop software around it like we did with elrs.

**Comment by ziiw** [comment]
What’s the difference we’re talking about?

**Reply by brandonrc** [reply]
![image](https://user-images.githubusercontent.com/25802530/144726109-64caa674-f223-4835-81a1-5a3aa7b0b2ac.png)

With digital systems you need 25mbps and 50mpbs.. kinda like DJI. 

Analog I am not actually sure.. but LoRa is probably not the best option for video.. it can barley send 4 channels of resolution and telemetry :P

**Reply by Lalbabu2001** [reply]
DJI uses some specific hardware and software which is closed source , can we develop around our own defined hardware like we use g28 chip for elrs.
